src/braket/simulator/braket_simulator.py

```diff
@@ -59,6 +62,56 @@ def run(
             representing the results of the simulation.
         """
 
+    def run_multiple(
+        self,
+        payloads: Sequence[Union[OQ3Program, AHSProgram, JaqcdProgram]],
+        args: Optional[Sequence[Sequence[Any]]] = None,
+        kwargs: Optional[Sequence[Mapping[str, Any]]] = None,
+        max_parallel: Optional[int] = None,
+    ) -> list[Union[GateModelTaskResult, AnalogHamiltonianSimulationTaskResult]]:
+        """
+        Run the tasks specified by the given IR payloads.
+
+        Extra arguments will contain any additional information necessary to run the tasks,
+        such as number of shots.
+
+        Args:
+            payloads (Sequence[Union[OQ3Program, AHSProgram, JaqcdProgram]]): The IR representations
+                of the programs
+            args (Optional[Sequence[Sequence[Any]]]): The positional args to include with
```

Comment:
Can I only submit **one** of `args` and `kwargs`? Is the intent that specifying `shots` in `kwargs` should override any value in `args`?

Reply:
You can specify both, just as you can technically
```python
device.run(circuit, 1000, shots=1000)
```
it'll just error out with something like
```
LocalSimulator.run() got multiple values for argument 'shots'
```